spec/features/ncr_work_orders_spec.rb

```diff
@@ -450,8 +440,8 @@
         click_on 'Update'
         proposal = Proposal.last
 
-        expect(proposal.approvers.first.email_address).to eq (approver.email_address)
-        expect(proposal.individual_approvals.first.actionable?).to eq (true)
+        expect(proposal.approvers.first.email_address).to eq approver.email_address
+        expect(proposal.individual_approvals.first.actionable?).to eq true
```

Comment:
... and in fact, we have use of such a matcher down the page. Didn't know about those - cool!